src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.scss

```diff
@@ -117,3 +117,45 @@ md-icon {
   color: $delicate;
   font-size: $subhead-font-size-base;
 }
+
+.kd-replicationcontrollerdetail-labeled-sparkline-label {
+  white-space: nowrap;
+  word-spacing: normal;
+}
+
+.kd-sparkline {
+  background-color: $body;
+  height: 2.5 * $baseline-grid;
+  vertical-align: -$baseline-grid / 2; // 20% below baseline, to flow with text
```

Comment:
@bryk - I've added the vertical-align back in. Without it, the graph visually looks too high, like it's a step up from the label text. If I change the enclosing line-height to zero, it moves the graph but also the label text, so the graph still looks higher than the mock. (I might not understand your line-height suggestion) - please let me know if this is ok.